### PR TITLE
C410760 - "User type" field is displayed on create user page (Poppy +) (Thunderjet) (TaaS)

### PR DESCRIPTION
**Pull request by tomleexx**
[Allure report]-https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5105/allure/
[C410760]- https://foliotest.testrail.io/index.php?/cases/view/410760
C410760-Allure-Report

### CODE REVIEW

File: cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    cy.login(userData.username, userData.password, {
+      path: TopMenu.usersPath,
+      waiter: UsersSearchPane.waitLoading,
+    });
+  });
+
+  after('Deleting created entities', () => {
+    cy.log(newUserId);
+    Users.deleteViaApi(userData.userId);
+    if (newUserId) {
```
Comment (manowarqqq) [inline cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js:64]: Is it not always new id?

**Review by manowarqqq — CHANGES_REQUESTED** [review]

File: cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      Users.deleteViaApi(newUserId);
+    }
+    PatronGroups.deleteViaApi(patronGroup.id);
+  });
+
+  it(
+    'C410760 "User type" field is displayed on create user page (Poppy +) (Thunderjet) (TaaS)',
```
[the remark this answers is not in the document]
Comment (tomleexx) [reply cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js:69]: https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5042/allure/

File: cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        ],
+        patronGroup.name,
+      ).then((userProperties) => {
+        userData = userProperties;
+        UserEdit.addServicePointViaApi(servicePointId, userData.userId, servicePointId);
+      });
+    });
+  });
+
+  before('Login', () => {
```
Comment (manowarqqq) [inline cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js:55]: Please combine 'before' to one

**Review by manowarqqq — COMMENTED** [review]
Please add screenshot with valid allure report to ensure test works after allure report will be expired
Please use rebase intead of merge and combine pr in one commit for future tests

**Review by manowarqqq — COMMENTED** [review]
-

File: cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js (hunk 3 above)
Comment (tomleexx) [reply cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js]: combine 'before' into one
then((userProperties) => {
        userData = userProperties;
        UserEdit.addServicePointViaApi(servicePointId, userData.userId, servicePointId);
        cy.login(userData.username, userData.password, {
          path: TopMenu.usersPath,
          waiter: UsersSearchPane.waitLoading,
        });

**Review by manowarqqq — APPROVED** [review]

**Comment by tomleexx** [comment]
> Please add screenshot with valid allure report to ensure test works after allure report will be expired Please use rebase intead of merge and combine pr in one commit for future tests

add screenshot in pull request comments

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**